src/Tribe/Views/V2/Hooks.php

```diff
@@ -566,6 +567,36 @@ public function manage_sensitive_info( $post ) {
 		}
 	}
 
+	/**
+	 * Fires to add JSON LD to Single Event
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function mange_single_json_ld() {
+
+		// Check if we are in a single page.
+		if ( ! is_singular( \Tribe__Events__Main::POSTTYPE ) ) {
+			return;
+		}
+
+		$context = tribe_context();
+
+		// One more check for our Post Type.
+		if ( ! $context->is( 'tec_post_type' ) ) {
+			return;
+		}
+
+		// Bail when that action already exists.
+		if ( has_action( 'wp_head', [ \Tribe__Events__JSON_LD__Event::instance(), 'markup' ] ) ) {
+			return;
+		}
+
+		// Print JSON-LD markup on the`wp_head`
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a19b318097325ed24202aae16467e3a2</sub>